src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.cs

```diff
@@ -52,15 +52,15 @@ public void TestWafDynamicManifest()
             TestRunner.RunTestScript("Test-WafDynamicManifest");
         }
 
-        [Fact]
+        [Fact(Skip = "Skip as creation of V1 gateways is no longer supported")]
```

Comment:
why is this test case skipped? There are no API upgrade or new API added, what is differences between v1 and v2?

Reply:
We have logic in the Network Resource Provider which now prevents the creation of V1 Application Gateways. Therefore, this test will always fail because creation is blocked.